specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimdeployment.json

```diff
@@ -1469,6 +1505,16 @@
       },
       "description": "Response of the CheckNameAvailability operation."
     },
+    "ApiManagementServiceGetDomainOwnershipIdentifierResult": {
```

Comment:
>ApiManagementServiceGetDomainOwnershipIdentifierResult [](start = 5, length = 54)

I assume this is treated as a secret, that's why you modeled this as a POST, if yes, add x-ms-secret as stated [here](https://armwiki.azurewebsites.net/rp_onboarding/process/property_design_best_practices.html#secrets)

Reply:
It's not used as a secret. It needs to be returned to customer to add as a Txt Record, then we use it to do Txt Record validation and avoid dangling domain issue.